### PR TITLE
Actually run storage integration tests

### PR DESCRIPTION
**Pull request by samtstern**
### Description

- Actually run the tests (they were not run in CI before)
- Add some tests to the storage emulator to verify some behavior exercised in import/export PR (#3317)

### Scenarios Tested

Uh, the PR is tests.

### Sample Commands

N/A

### CODE REVIEW

File: scripts/storage-emulator-integration/tests.ts [hunk 1]
```diff
@@ -244,6 +244,20 @@ describe("Storage emulator", () => {
         });
       });
 
+      describe("#get()", () => {
```
Comment (samtstern) [inline scripts/storage-emulator-integration/tests.ts:247]: This test is failing with an admin credential error? I don't understand why/how

File: scripts/storage-emulator-integration/tests.ts [hunk 2]
```diff
@@ -244,6 +246,20 @@ describe("Storage emulator", () => {
...
```
Comment (samtstern) [inline scripts/storage-emulator-integration/tests.ts:249]: There is a failure in our metadata, this errors out due to a `crc32c` mismatch.

**Comment by abeisgoat** [comment]
The tests not running was intentional since our Cloud Build pipeline doesn't currently have a `java` install, however the crc32c mismatch is not intentional.

**Comment by abeisgoat** [comment]
@googlebot I consent.

File: src/emulator/storage/server.ts [hunk 3]
```diff
@@ -63,6 +63,11 @@ export function createApp(
     })
   );
 
+  app.get("/internal/export", (req, res) => {
```
Comment (samtstern) [inline src/emulator/storage/server.ts:66]: This seems like a mistake?

Comment (abeisgoat) [reply src/emulator/storage/server.ts:66]: yes

**Review by abeisgoat — APPROVED** [review]

**Pull request merged**